packages/core/src/__tests__/ceramic-feed.test.ts

```diff
@@ -1,44 +1,40 @@
 import { expect, describe, test, beforeEach, afterEach } from '@jest/globals'
-import { TestUtils, type StreamState, type IpfsApi } from '@ceramicnetwork/common'
+import { TestUtils, type IpfsApi } from '@ceramicnetwork/common'
 import { TileDocument } from '@ceramicnetwork/stream-tile'
-import { createIPFS, swarmConnect, withFleet } from '@ceramicnetwork/ipfs-daemon'
+import { createIPFS, swarmConnect } from '@ceramicnetwork/ipfs-daemon'
 import type { Ceramic } from '../ceramic.js'
-import { createCeramic as vanillaCreateCeramic } from './create-ceramic.js'
 import { Model, ModelDefinition } from '@ceramicnetwork/stream-model'
 import { take } from 'rxjs'
-import { Document } from '../feed.js'
-
-function createCeramic(
-  ipfs: IpfsApi,
-  anchorOnRequest = false,
-  streamCacheLimit = 100
-): Promise<Ceramic> {
-  return vanillaCreateCeramic(ipfs, {
-    anchorOnRequest,
-    streamCacheLimit,
-  })
-}
+import { FeedDocument } from '../feed.js'
+import { createCeramic } from './create-ceramic.js'
 
 describe('Ceramic feed', () => {
-  let ipfs: IpfsApi
-  let ceramic: Ceramic
+  let ipfs1: IpfsApi
+  let ipfs2: IpfsApi
+  let ceramic1: Ceramic
+  let ceramic2: Ceramic
   beforeEach(async () => {
```

Comment:
can we make this beforeAll/afterAll?  Do we need the nodes spun up and down between each test?  Not a big deal either way, but if we can re-use the same nodes for the whole test it will improve test execution speed.

Reply:
Good point, ready!